de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/LocalEditorManipulator.java

```diff
@@ -57,7 +60,7 @@ public void initialize(EditorManager editorManager) {
      * or <code>null</code> if the file does not exist or is not shared
      */
     public Editor openEditor(SPath path, boolean activate) {
-        if (!manager.getSession().isShared(path.getResource())) {
+        if (!SessionUtils.isShared(path.getResource())) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "SessionUtils"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=293&issues=AWcxWGhu0POp3Yiz0EjG&open=AWcxWGhu0POp3Yiz0EjG)

[ISSUE_KEY]: AWcxWGhu0POp3Yiz0EjG